### DISCUSSION TITLE
What Dockerfile that used in Dockerhub

### DISCUSSION THREAD

**Discussion by prakashdivyy** [chatwoot/chatwoot#3018] [Q&A]
Hello, I want to ask about which Dockerfile that used in [Dockerhub](https://hub.docker.com/layers/chatwoot/chatwoot/release-v1.20.0/images/sha256-ebcde8f8e6615854200ce2e4456247c90421655f85696b71031c56a4d5f5ef4c?context=explore)?

I want to create a custom image, so that I can use it into [chatwoot helm chart](https://www.chatwoot.com/docs/self-hosted/deployment/helm-charts)

Thanks!

**Comment by pranavrajs** [comment]
@prakashdivyy Please see this file. https://github.com/chatwoot/chatwoot/blob/develop/docker/Dockerfile
